SECURITY.md

```diff
@@ -0,0 +1,82 @@
+# Security Policy
+
+NOTE: Currently, this policy is more for the developers rather the end users. This will however change when there is a MVP and the application is available to users. 
+
+If you are going to contribute to this project, please take some time out to read through this. Follow the set rules and NO exceptions. 
+
+We focus on the following principles:
+
+1. Validate input.
+2. Keep it simple.
+3. Default deny.
+4. Adhere to the principle of least privilege.
+5. Sanitize data sent to other systems.
+6. Only store data that you need. 
+7. PII and financial data MUST be encrypted
+
+## Reporting a Vulnerability
+
+To report a vulnerability, create a new topic in the Issues. Be straight to the point and precise. 
```

Comment:
We need to discuss how to do vuln reporting. It'd be better to take a Project Zero approach: privately disclose the vuln, and wait 60 days for either a patch or response that the vuln is being addressed, and if after 60 days there's no response, then report publicly